.changeset/little-boats-think.md

```diff
@@ -0,0 +1,9 @@
+---
+'@backstage/plugin-catalog-react': minor
+---
+
+`EntityOwnerPicker` now loads entities asynchronously
+
+**BREAKING**: In order to improve the performance of the component, the users and groups displayed by `EntityOwnerPicker` aren't inferred
```

Comment:
Can't really both label this as breaking and also just do a minor bump 😅. Tbh I think it is alright to call this a non-breaking behavior change, since the user experience is similar and there's no API change.